plugin.rb

```diff
@@ -186,19 +196,58 @@ class ::TopicListItemSerializer
   require_dependency 'list_controller'
   class ::ListController
     generate_message_route(:private_messages_assigned)
-    generate_message_route(:messages_assigned)
   end
 
   add_to_class(:topic_query, :list_messages_assigned) do |user|
-    list = joined_topic_user.where("
+    list = default_results.where("
       topics.id IN (
         SELECT topic_id FROM topic_custom_fields
         WHERE name = 'assigned_to_id'
         AND value = ?)
     ", user.id.to_s)
       .order("topics.bumped_at DESC")
 
-    create_list(:assigned, {}, list)
+    return create_list(:assigned, {}, list)
```

Comment:
No need to have `return` here in Ruby

(I think rubocop will actually fail for this)